auth-jwt-service/src/main/java/io/stargate/auth/jwt/JwtValidator.java

```diff
@@ -0,0 +1,78 @@
+package io.stargate.auth.jwt;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.jwk.source.JWKSource;
+import com.nimbusds.jose.jwk.source.RemoteJWKSet;
+import com.nimbusds.jose.proc.BadJOSEException;
+import com.nimbusds.jose.proc.JWSKeySelector;
+import com.nimbusds.jose.proc.JWSVerificationKeySelector;
+import com.nimbusds.jose.proc.SecurityContext;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.proc.ConfigurableJWTProcessor;
+import com.nimbusds.jwt.proc.DefaultJWTProcessor;
+import io.stargate.auth.UnauthorizedException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.text.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JwtValidator {
```

Comment:
`JwtValidator` doesn't need to be a class. This logic can just be inside of `AuthJwtService`.

Reply:
Yeah, I guess I can just pass AuthJWTService a `ConfigurableJWTProcessor`